lib/browser/methods/platform.rb

```diff
@@ -35,6 +35,16 @@ def ios8?
       ios? && !!(ua =~ /OS 8/)
     end
 
+    # Detect if is iOS app webview.
+    def ios_app?
+      ios? && !ua.include?('Safari')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.